assets/css/frontend/base/wordpress.css

```diff
@@ -0,0 +1,31 @@
+/* WordPress generated classes */
+
+.alignleft {
+	float: left;
+	margin-right: 1rem;
+}
+
+.alignright {
+	float: right;
+	margin-left: 1rem;
+}
+
+.aligncenter {
```

Comment:
Don't we need both options for this? `.aligncenter, img.aligncenter` (and the others)

Reply:
@timwright12, I don't think it's needed. I've seen it one some projects but don't see the need.

![screen shot 2018-10-09 at 9 30 37 am](https://user-images.githubusercontent.com/2092765/46673375-9e437c80-cba7-11e8-9bfa-50c8e9ca8dd0.png)

Reply:
I ran into this issue on a project recently. When alignment is assigned to an image, and you shrink the browser size, the text will begin to flow around the image, which is great, however it gets to a point where there are only 2 or 3 words per line.

Is there some sort of responsive fallback we can add here? Perhaps at a certain breakpoint, any aligned images should just center themselves so that we avoid extremely short line lengths?

Who would have thought a simple base set of WP styles would yield such a lengthy PR. You rock for rolling through this with us @smy315, thank you!

Reply:
@magnificode those 2-3 words per line because of image alignment are annoying but... how about for small images that should remain aligned left or right? I think it will be hard to account of al possible image sizes. Take a look at this screengrab: 

![images](https://user-images.githubusercontent.com/2092765/46876578-aea86100-ce0c-11e8-8853-18d986c6f818.gif)

That 150 by 150 image might look off centered.

Feels like deciding to center images on smaller screen should be a project-based decision with setting media image sizes and breakpoints.

Reply:
Totally fair point @smy315 - thanks so much!